src/main/java/frc/robot/Constants.java

```diff
@@ -22,8 +22,8 @@ public static final class OperatorConstants {
     public static final int OPERATOR_CONTROLLER_PORT = 1;
   }
 
-  public static final class EngancheConstants {
-    public static final int CAN_ID = 7;
+  public static final class HangingConstants {
+    public static final int CAN_ID = 8;
```

Comment:
```suggestion
    public static final int CAN_ID = 7;
```

El enganche debe de ser ID 7